cmd/db_audit_logs_exporter/db_audit_logs_exporter.go

```diff
@@ -201,8 +201,8 @@ var removeDbAuditLogsExporterCmd = &cobra.Command{
 func init() {
 	DbAuditLogsExporterCmd.AddCommand(assignDbAuditLogsExporterCmd)
 	assignDbAuditLogsExporterCmd.Flags().SortFlags = false
-	assignDbAuditLogsExporterCmd.Flags().String("telemetry-provider-id", "", "[REQUIRED] The ID of the telemetry provider")
-	assignDbAuditLogsExporterCmd.MarkFlagRequired("telemetry-provider-id")
+	assignDbAuditLogsExporterCmd.Flags().String("integration-id", "", "[REQUIRED] The ID of the integration")
```

Comment:
Not sure but should this text value **I**ntegration with caps I instead of i, WDYT

Reply:
`[REQUIRED] The ID of the Integration` this part?
the commands are case insensitive afaik `integration-id`

Reply:
yes - "The ID of the Integration this part"